packages/frontend/src/components/Editor/SlateInputField.tsx

```diff
@@ -32,7 +32,7 @@ export const SlateInputField = () => {
         editor={editor}
         value={slateValue}
         onChange={(value: any) => {
-          const isAstChange = editor.operations.some((op) => 'set_selection' !== op.type);
+          const isAstChange = editor.operations.some((op: any) => 'set_selection' !== op.type);
```

Comment:
Any (hehe) chance to get rid of this any? And even on the value one line above as well. Not necessary, but would be nice.